internal/remote/syscall_dup_unix.go

```diff
@@ -0,0 +1,9 @@
+// +build !linux !arm64
```

Comment:
Yes, of course! It's a bit hard to test on all platforms :)
Now the formula is:
`((NOT linux) OR (NOT arm64)) AND (NOT windows)`
and that's equalient to 
`(NOT linux,arm64) AND (NOT windows)`

Now it works on windows too...